style.css

```diff
@@ -0,0 +1,363 @@
+a {
+  text-decoration: none;
+}
+header {
+  background-color: black;
+  display: none;
+  width: 100%;
+  height: 25px;
+}
+body {
+  margin: 0px;
+}
+.white {
+  color: white;
+}
+.headerRight {
+  float: right;
+}
+header a {
+  color: rgb(141, 141, 132);
+  font-family: "ARS Maquette", "Helvica Neue", Arial, sans-serif;
+  font-size: 13px;
+}
+.divLinks {
+  display: none;
+  flex-grow: 4;
+}
+.divLinks a {
+  color: rgb(170, 170, 170);
+  font-family: Shentox, sans-serif;
+  font-size: 14px;
+  padding: 18px 12px 0px 15px;
+}
+.divIcon {
+  position: relative;
+  top: 17px;
+  right: 15px;
+}
+.divIcon a {
+  height: 48px;
+}
+main {
+  clear: both;
+}
+.TNWWhiteLogo {
+  background-color: black;
+  width: 100px;
+  padding: 10px;
+}
+.fa-bars {
+  color: black;
+  height: 48px;
+}
+.fa-search {
+  color: black;
+  height: 48px;
+}
+.headCont {
+  min-height: 200px;
+  display: flex;
+  margin: 10px;
+  flex-direction: column-reverse;
+}
+.hiddenLogo {
+  width: 50%;
+  margin: 15px;
+  display: none;
+}
+.articleHeader {
+  margin: auto 4px 0px 4px;
+  position: relative;
+}
+nav {
+  height: 48px;
+  display: inline-flex;
+  width: 100%;
+  justify-content: space-between;
+}
+.header1Cont {
+  background-image: url("Images/header1.png");
+}
+.header2Cont {
+  background-image: url("Images/header2.jpg");
+}
+.header3Cont {
+  background-image: url("Images/header3.jpg");
+}
+.orangeTitle {
+  color: #f42;
+  font-family: Avalon, sans-serif;
+  font-size: 28px;
+}
+.orangeText {
+  color: #f42;
+}
+.latestFlex {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0px 5px 0px 10px;
+}
+.latest {
+  display: flex;
+  width: 100%;
+  margin: 5px 0px;
+}
+.latest img {
+  height: 75px;
+  width: 75px;
+}
+.credits {
+  color: rgba(141, 141, 141, 0.83);
+  font-family: "ARS Maquette", "Helvica Neue", Arial, sans-serif;
+  font-size: 12px;
+}
+.latestTitle {
+  font-family: "ARS Maquette", "Helvica Neue", Arial, sans-serif;
+  font-size: 13.5px;
+  font-weight: bold;
+  margin: 0px;
+}
+.latestText {
+  margin: 0 5px;
+}
+.funlog {
+  height: 60px;
+  width: 60px;
+  float: left;
+  margin: 5px;
+}
+.latfundCont {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 5px 15px;
+}
+.lafund {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 10px 0px;
+}
+.lafundHead {
+  width: 100%;
+}
+.sect {
+  width: 100%;
+  display: flex;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```CSS
.flex{
  display: flex;
}
```
**Html file**

```HTML
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_